### PR TITLE
Merge changes from #13592 to develop

### PR DESCRIPTION
**Pull request by psteckler**
Allow clean merge-to-develop for #13592.

### CODE REVIEW

**Comment by psteckler** [comment]
!ci-build-me

**Comment by psteckler** [comment]
!ci-build-me

**Review by ghost-not-in-the-shell — APPROVED** [review]

File: src/lib/crypto/snarky_tests/examples/range_gt.json [hunk 1]
```diff
@@ -1 +1 @@
...
...row":6,"col":6}],"coeffs":["0000000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","0100000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040","0000000000000000000000000000000000000000000000000000000000000000","0100000000000000000000000000000000000000000000000000000000000000"]},{"typ":"Generic","wires":[{"row":6,"col":2},{"row":7,"col":1},{"row":7,"col":2},{"row":7,"col":3},{"row":7,"col":4},{"row":7,"col":5},{"row":7,"col":6}],"coeffs":["0100000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","00000000ed302d991bf94c09fc98462200000000000000000000000000000040"]}]}
```
Comment (querolita) [inline src/lib/crypto/snarky_tests/examples/range_gt.json:1]: In these files, is it OK that coefficients appear as hex strings whereas the same file in #13592 are given as arrays of bytes?

**Review by querolita — APPROVED** [review]
Left one question 👍🏻

File: src/lib/crypto/snarky_tests/examples/range_gt.json (hunk 1 above)
Comment (psteckler) [reply src/lib/crypto/snarky_tests/examples/range_gt.json:1]: The updates to the examples JSON are displayed when you run the unit tests, explaining the difference in format. 

Using the same format as in #13592 results in errors.

**Pull request merged**